### ISSUE TITLE
clearer display of cei:app in single charter view

### ISSUE THREAD

**Issue by NTsch** [icaruseu/mom-ca#1086]
Currently, the `cei:app` element is displayed by simply outputting its text values, so that, for example, the following:
` <cei:app>
                     <cei:lem wit="#a">in perpetuum</cei:lem>
                     <cei:rdg wit="#B">in pontificatu pontifex maximus</cei:rdg>
                  </cei:app>`
... is written as _in perpetuum in pontificatu pontifex maximus_ (see https://www.monasterium.net/mom/Fontenay/Fontenay_170/imported-charter). This does not make it clear that this is an apparatus, and a clearer display has been requested.

**Comment by GVogeler** [comment]
my suggestion would be a tooltip starting with a xrx:message for "variant readings:" and listing the content of each `rdg`:
```xml
<xsl:for-each select="rdg">
<xsl:apply-templates select="*"/> <span style="font-weight:bold"><xsl:value-of select="@wit"></span>
</xsl:for-each>
```

**Comment by NTsch** [comment]
> my suggestion would be a tooltip starting with a xrx:message for "variant readings:" and listing the content of each `rdg`:
> 
> ```
> <xsl:for-each select="rdg">
> <xsl:apply-templates select="*"/> <span style="font-weight:bold"><xsl:value-of select="@wit"></span>
> </xsl:for-each>
> ```
Yes, this is along the lines of what I currently have:
![Screenshot from 2022-08-30 10-03-56](https://user-images.githubusercontent.com/43009273/187384840-0f1d7cc2-133b-40fb-8fbc-18e2b70402cb.png)

**Comment by NTsch** [comment]
Closed by #1087.
